pyproject.toml

```diff
@@ -30,6 +30,8 @@ classifiers = [
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
 space_packet_parser = ">=4.0.2,<5"
+pandas = ">=2.0.0,<3"
+openpyxl = ">=3.0.7,<4"
```

Comment:
If you like to make changes based on Greg's comment, you will need to remove those two from here and add to optional. Suggested down below.
```suggestion
```